mailbox/api/src/main/java/org/apache/james/mailbox/model/SimpleMailboxAnnotation.java

```diff
@@ -0,0 +1,328 @@
+package org.apache.james.mailbox.model;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.regex.Pattern;
+
+import org.apache.james.mailbox.exception.UnsupportedAnnotationException;
+import org.apache.james.mailbox.model.MailboxACL.MailboxACLEntryKey;
+
+import com.google.common.base.Preconditions;
+
+public class SimpleMailboxAnnotation implements MailboxAnnotation {
+
+    private static final Pattern NAME_ANNOTATION_PATTERN = Pattern.compile("[^\\x1a-\\x7f]");
+
+    public static final MailboxAnnotation EMPTY;
+
+    private final Map<MailboxAnnotationEntryKey, MailboxAnnotationEntryValue> entries;
+
+    static {
+        EMPTY = new SimpleMailboxAnnotation();
+    }
+
+    /**
+     * Creates a new instance of SimpleMailboxAnnotation containing no entries.
```

Comment:
it looks like a useless comment. It would be way better to have a static method `empty()` that returns an empty instance.